Sources/NIOHTTP2/InboundWindowManager.swift

```diff
@@ -67,7 +71,16 @@ struct InboundWindowManager {
         return self.calculateWindowIncrement(windowSize: lastWindowSize + self.bufferedBytes)
     }
 
+    mutating func close() {
```

Comment:
I don't think the indirection is needed; lets just have `closed` be an internal property on the window manager and set it directly.